ehitchhiking-ui/src/app/app.component.sass

```diff
@@ -1 +1,14 @@
+html,
+body,#map
+  width: 100%
+  height: 100%
+  padding: 0
+  margin: 0
+  overflow: hidden
+  position: absolute
```

Comment:
It is a bad practice to set position absolute to html, body. Remove it

Reply:
remove position